kumascript/macros/WebExtAPISidebar.ejs

```diff
@@ -36,72 +38,74 @@ async function buildSublist(pages, title, ignoreBadges) {
       apiComponentName = titlePieces[titlePieces.length - 1];
     }
 
-    if (locale != 'en-US') {
-        aPage.translations.forEach(function(translation) {
-            if(translation.locale === locale) {
-                url = translation.url;
-                title = htmlEscape(translation.title);
-            }
-        });
+    if (locale != "en-US") {
+      aPage.translations.forEach(function (translation) {
+        if (translation.locale === locale) {
+          url = translation.url;
+          title = htmlEscape(translation.title);
+        }
+      });
     }
 
-    result += '<li>';
+    result += "<li>";
 
-    let pageBadges = '';
+    let pageBadges = "";
     if (!ignoreBadges) {
-        pageBadges = (await page.badges(aPage)).join("");
+      pageBadges = (await page.badges(aPage)).join("");
     }
 
     if (rtlLocales.indexOf(locale) != -1) {
-        result += '<bdi>';
+      result += "<bdi>";
     }
 
     if (slug == aPage.slug) {
-        result += `<em><code>${apiComponentName}</code> ${pageBadges}</em>`;
+      result += `<em><code>${apiComponentName}</code> ${pageBadges}</em>`;
     } else {
-        result += `<a href="${url}"><code>${apiComponentName}</code></a>${pageBadges}`;
+      result += `<a href="${url}"><code>${apiComponentName}</code></a>${pageBadges}`;
     }
 
     if (rtlLocales.indexOf(locale) != -1) {
-        result += '</bdi>';
+      result += "</bdi>";
     }
 
-    result += '</li>';
+    result += "</li>";
   }
 
-  result += '</ol></li>';
+  result += "</ol></li>";
 
   return result;
 }
 
 function classifyPages(subpages) {
-  var collection = {
-      properties: [],
-      methods: [],
-      events: [],
-      types: []
-  }
-
-  subpages.forEach(function(subpage) {
-    if (hasTag(subpage, "Property")) {
-      collection.properties.push(subpage);
-    }
-    if (hasTag(subpage, "Method")) {
-      collection.methods.push(subpage);
-    }
-    if (hasTag(subpage, "Type")) {
-      collection.types.push(subpage);
-    }
-    if (hasTag(subpage, "Event")) {
-      collection.events.push(subpage);
+  const collection = {
+    properties: [],
+    methods: [],
+    events: [],
+    types: [],
+  };
+
+  for (const subpage of subpages) {
+    switch (subpage.pageType) {
+      case "webextension-api-property":
+        collection.properties.push(subpage);
+        break;
+      case "webextension-api-function":
+        collection.methods.push(subpage);
+        break;
+      case "webextension-api-type":
+        collection.types.push(subpage);
+        break;
+      case "webextension-api-event":
+        collection.events.push(subpage);
+        break;
```

Comment:
Unfortunately, the reformatting makes it hard to see what has actually been changed.

@wbamberg Is this the main change of this PR? And if so, could we revert the other reformatting changes?

Reply:
Because this PR included Prettier I made this PR in two commits.

- the Prettier change in a separate commit: https://github.com/mdn/yari/commit/ca999664999a4ae3b8026023afbbc73cf170161e.

- the substantive, non-formatting change is in https://github.com/mdn/yari/commit/611a0e54dc256c56ab1f7b921953b6f3b49f2a1c.

I did say this in the PR description.

But if it's not OK to include the Prettier change I can revert it.

Reply:
No, that's okay, sorry for missing that part of the PR description.